doc/monitoring-stack-deployment/README.md

```diff
@@ -10,9 +18,11 @@ spec:
 ```
 2. Install Prometheus operator v4.10.0 from the Operator Hub.
 
-3. Install Grafana operator v4.10.1 from the Operator Hub.
+#### OpenShift versions up to and including 4.15
+
+1. Install Grafana operator v4.10.1 from the Operator Hub.
```

Comment:
OSD 4.15 allow installation of Grafana v4.
Seems to me need write it differ . 
The difference - missing secret with THANOS_QUERIER_BEARER_TOKEN in 4.16, so instead of searching the Token  - need create it.
It's not related to version of Grafana

Reply:
how we set this up is related to openshift version. As the title of this section "OpenShift versions up to and including 4.15".
Any suggestions?

Reply:
Yes, I see you did 2 different sections for v4 and v5, where each one contains All steps.
I think it's Good and very convenient for reader. Thank you.